python/sglang/srt/sampling_params.py

```diff
@@ -123,3 +123,17 @@ def normalize(self, tokenizer):
                 else:
                     stop_str_max_len = max(stop_str_max_len, len(stop_str))
             self.stop_str_max_len = stop_str_max_len
+
+    def to_srt_kwargs(self):
+        return {
+            "max_new_tokens": self.max_new_tokens,
+            "stop": self.stop_strs,
```

Comment:
Does `stop_strs` need to be processed as a list, in the case of multiple stop str scenarios?

Reply:
stop is either str or list of str, both type can be handled by json.dumps. the token_ids is a set which need to be converted to list.